website_menu_url_translate/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 initOS GmbH. <http:/www.initos.com>
+# Copyright 2017 GYB IT SOLUTIONS <http:/www.gybitsolutions.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Website Menu URL Translation",
+    'category': "Website",
+    'website': "www.gybitsolutions.com",
```

Comment:
@rami-wafaie Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)